plugins/inputs/webhooks/particle/particle_webhooks_test.go

```diff
@@ -0,0 +1,97 @@
+package particle
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func postWebhooks(rb *ParticleWebhook, eventBody string) *httptest.ResponseRecorder {
+	req, _ := http.NewRequest("POST", "/", strings.NewReader(eventBody))
+	w := httptest.NewRecorder()
+	w.Code = 500
+
+	rb.eventHandler(w, req)
+
+	return w
+}
+
+func TestNewItem(t *testing.T) {
+	t.Parallel()
+	var acc testutil.Accumulator
+	rb := &ParticleWebhook{Path: "/particle", acc: &acc}
+	resp := postWebhooks(rb, NewItemJSON())
+	if resp.Code != http.StatusOK {
+		t.Errorf("POST new_item returned HTTP status code %v.\nExpected %v", resp.Code, http.StatusOK)
+	}
+
+	fields := map[string]interface{}{
+		"temp_c":    26.680000,
+		"temp_f":    80.024001,
+		"infrared":  528.0,
+		"lux":       0.0,
+		"humidity":  44.937500,
+		"pressure":  998.998901,
+		"altitude":  119.331436,
+		"broadband": 1266.0,
+	}
+
+	tags := map[string]string{
+		"id":       "230035001147343438323536",
+		"location": "TravelingWilbury",
+	}
+
+	acc.AssertContainsTaggedFields(t, "temperature", fields, tags)
+}
+
+func TestUnknowItem(t *testing.T) {
```

Comment:
typo: TestUnknowItem -> TestUnknownItem

Reply:
Typo carried over from the Rollbar plugin. I thought maybe it was historically required. :-)

Reply:
lol